DigitalExcellence/dex-frontend
# Bugfix/195 highlight responsive

## Description

Make the highlight cards responsive for different screen sizes and long titles will cut off and put on the second line . On smaller viewports the cards will be placed below each other. On large viewports, the cards will have a fixed height, this is because all the cards should have the same height. But on smaller screenports, the description will have a fixed height. This is because whenever the title cuts off to a second line, the description can be pushed out of the card. To make sure this didn't happen, the description has a fixed height. This results in different heights for the cards, but this isn't a problem for this viewport because the cards are below each other.


## Type of change


## Checklist


## Steps to Test or Reproduce

1. Check the highlight card for a long name, so it has to be cut of over multiple lines
2. Change the screen size to 991px wide. The cards should stack below each other with some space between the cards.
3. Change the screen size to 767px wide. The image and the title should be centered below each other.
4. Change the screen size to 575px wide. The description height should be set to the fixed 110px (or lower for shorter descriptions).

## Link to issue

Closes: #195
